plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammar_fstrings/build.xml

```diff
@@ -1,7 +1,11 @@
 <project name="jython-parser" default="parser">
+	
+	
 
-    <property name="javaccHome" value="c:\bin\javacc-5.0" />
-    <property name="python" value="c:\bin\Miniconda\envs\py27_tests\python.exe" />
+	<property environment="env"/>
+	<property name="userProfile" value="${env.USERPROFILE}"/>
+    <property name="javaccHome" value="${userProfile}\Documents\Javacc\javacc-5.0" />
```

Comment:
The paths are still hardcoded... the idea is having 2 (made up) variables without anything hardcoded (say, GRAMMAR_JAVACC_DIR and GRAMMAR_PYTHON_EXE).